controllers/channels.js

```diff
@@ -80,21 +81,21 @@ module.exports = (app, passport, React, ReactDOMServer) => {
       <br />
       Description:
       ${data.description}
-    `
+    `;
     transporter.sendMail({
       subject: `Report user: ${data.message.sender}`,
       html: body,
-      to: "info+report-a-user@sigwo.com",
-      from: process.env.EMAIL
+      to: 'info+report-a-user@sigwo.com',
```

Comment:
Email should come from env vars.

Reply:
Yeah, I agree, Luis added this email, my IDE changed the "" to '' that's why we have this change on my PR